apps/store/src/components/PriceCalculator/useHandleSubmitPriceCalculator.ts

```diff
@@ -28,6 +28,7 @@ export const useHandleSubmitPriceCalculator = (params: Params) => {
       }
     },
     onError(error) {
+      window.console.log("Couldn't update price intent", error)
```

Comment:
Auto completion maybe? 🤔 
```suggestion
      console.log("Couldn't update price intent", error)
```